bool jump_now

---

string message

bool success
